Search/Response+Parsing.swift

```diff
@@ -39,28 +39,31 @@ private func parseResponse(data: NSData?) -> Result<Response> {
     guard let data = data else {
         return .Failure(SearchError.NoDataReceived)
     }
-    guard let json = JSValue.parse(data).value else {
+    guard let responseJson = OSJSON(data: data) else {
         return .Failure(SearchError.FailedToParseJSON)
     }
-    guard let response = responseFromJson(json) else {
+    guard let response = responseFromOSJson(responseJson) else {
         return .Failure(SearchError.FailedToDeserialiseJSON)
     }
     return .Success(response)
 }
 
-private func responseFromJson(json: JSON) -> Response? {
-    let headerJson = json[Response.HeaderKey]
-    guard let header = Header(json: headerJson) else {
-        return nil
+private func responseFromOSJson(json: OSJSON) -> Response? {
```

Comment:
In fact the template has a version of this already, and then you can bin the `dpaFromOSJson` function as well.